package.json

```diff
@@ -84,6 +84,9 @@
     "tsify": "^5.0.4",
     "typescript": "^4.6.4"
   },
+  "publishConfig": {
+    "registry": "https://slack.jfrog.io/slack/api/npm/npm-registry/"
```

Comment:
Actually one thought, is this URL something that we in any way want to be private?  At this point anyone could find it on github

Reply:
I suspect not, but good question. Asking around.